### ISSUE TITLE
derivatives rates and slopes - training: Problems 4 and 5

### ISSUE THREAD

**Issue by weshuu** [ofloveandhate/calculus_1#37]
In problem 4, the last question is:

"At which points must f be discontinuous? Use a comma-separated list of points."

It wants all x-values where f' is discontinuous to be listed. However:

- One of them is a jump discontinuity for f'; it's possible f is continuous with a "corner" at that point.
- One of them is an infinite discontinuity, with f' approaching -infinity from BOTH sides. It could be that f is shaped the way -x^(1/3) is shaped near the origin (with a vertical tangent line).
- In problem 5, there's an infinite discontinuity with f' approaching infinity from one side and -infinity from the other. I think f can still be continuous here, by having a corner *and* a vertical "tangent" line (picture y=sqrt(|x|)).
- Both problems also have removable discontinuities. I *believe* it's impossible for f to be continuous in these spots, but reasoning through this seems like a lot to ask of the students.

Did you intend to show a graph of f and ask where f' is continuous?

**Comment by ofloveandhate** [comment]
You're correct, this problem is flawed.  Thank you so much for noticing and putting in the thought to show how it's flawed!  Ultimately, I want a problem set that covers what you so beautifully described above. 

In the meantime, I wonder: what we should do with this problem?

**Comment by ofloveandhate** [comment]
I kind of want to move it from this set into the proving grounds.  I don't think the problem can be saved without a bunch of effort, which I cannot put in right now.

**Comment by ofloveandhate** [comment]
I think the right thing to do right now is just say the given graph is f, and ask where f' doesn't exist.  I'm going to do that for these two problems, and then copy into the other instructors' courses.

**Comment by ofloveandhate** [comment]
I went ahead and changed these two problems to make them mathematically correct, for all instructors

**Comment by ofloveandhate** [comment]
i'm re-opening this because I really want to make a set exploring derivatives of piecewise functions and other discontinuous functions

**Comment by weshuu** [comment]
There is also at least one problem in the Challenge Set (or the corresponding group) with the same issue.
